### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by AnkitPatanaik**
Make Registration Redux friendly.

### CODE REVIEW

File: src/containers/LoginView.jsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ class LoginView extends React.Component {
     return (
       <div className="tile is-child box">
         <h1 className="title">Login</h1>
-        <form onSubmit={this.loginUser}>
+        <form>
```
Comment (jay-hankins) [inline src/containers/LoginView.jsx]: Why did you change this?

Comment (AnkitPatanaik) [reply src/containers/LoginView.jsx]: it doesn't do anything

Comment (jay-hankins) [reply src/containers/LoginView.jsx]: I think leaving it out means that the Enter/Return key doesn't work to submit the form. Can you try both and confirm?

Comment (AnkitPatanaik) [reply src/containers/LoginView.jsx]: yep still works

File: src/actions/registrationActions.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ function registrationFailure() {
   return { type: types.REGISTER_FAILURE };
 }
 
-export function registerUser(credentials) {
+export default function registerUser(credentials) {
```
Comment (jay-hankins) [inline src/actions/registrationActions.js]: 👍
